.codecov.yml

```diff
@@ -0,0 +1,13 @@
+codecov:
+  notify:
+    require_ci_to_pass: no
+    after_n_builds: 1
```

Comment:
If you want to delay reporting until all builds are finished [see here](https://github.com/pysal/splot/issues/124#issue-782916169).